frontend/src/components/dashboard/components/categories-list/components/category/category.tsx

```diff
@@ -12,13 +13,33 @@ type Props = {
 };
 
 const Category: FC<Props> = ({ keyName, name }) => {
+  const { searchParams, performSearch } = useSearch();
+  const activeCategory = searchParams.get('category') || '';
```

Comment:
Is not "??" operator more suitable here? I do not think that category can be 0 (because logical OR operates with all falsy values (like '', 0, false, null, undefined or NaN) and will use the value on the right side of the operator (seems redundant in case of "searchParams.get('category') || ''")). At that very time nullish coalescing operator (??) takes falsy only null or undefined values